rosbag2_cpp/src/rosbag2_cpp/writers/sequential_writer.cpp

```diff
@@ -55,31 +57,63 @@ SequentialWriter::SequentialWriter(
   storage_(nullptr),
   metadata_io_(std::move(metadata_io)),
   converter_(nullptr),
+  compressor_(nullptr),
```

Comment:
If you're adding this, you should initialize it using braces.
Ignore this comment otherwise